packages/@o3r/eslint-plugin/package.json

```diff
@@ -35,18 +37,20 @@
     "eslint": "^8.22.0"
   },
   "devDependencies": {
+    "@angular-devkit/core": "~15.1.3",
+    "@angular-devkit/schematics": "~15.1.3",
     "@angular-eslint/eslint-plugin": "~15.2.0",
     "@angular-eslint/template-parser": "~15.2.0",
     "@angular-eslint/utils": "~15.2.0",
     "@angular/compiler": "~15.1.2",
     "@nrwl/jest": "^15.6.3",
     "@o3r/build-helpers": "workspace:^",
-    "@o3r/eslint-config-otter": "workspace:^",
```

Comment:
this is not mandatory for the linter to run?

Reply:
as you have seen on the CI it is not. Our .eslintrc files from workspaces are extending the file from root which is using the package.